src/lexedata/exporter/phylogenetics.py

```diff
@@ -57,11 +57,13 @@ def read_cldf_dataset(
 
     Examples
     --------
-
-    >>> _size = open("forms.csv", "w").write('''
+    >>> import tempfile
+    >>> dirname = Path(tempfile.mkdtemp(prefix="lexedata-test"))
+    >>> target = dirname / "forms.csv"
+    >>> _size = open(target, "w").write('''
     ... ID,Language_ID,Parameter_ID,Form,Cognateset_ID
     ... '''.strip())
-    >>> ds = pycldf.Wordlist.from_data("forms.csv")
+    >>> ds = pycldf.Wordlist.from_data(target)
```

Comment:
Nice changes.
For this kind of thing, maybe we do want pyfakefs, because it is much less boilerplate?

Reply:
So far we only have this doctest, so I think we can live with the boilerplate.